addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -429,7 +462,7 @@
 
 <template id="modal_ticket_registration" name="Modal for tickets registration">
     <!-- Modal -->
-    <div class="modal fade" id="modal_ticket_registration" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-labelledby="staticBackdropLabel" aria-hidden="true">
+    <div class="modal fade" t-attf-id="modal_{{'slot_ticket' if event.is_multi_slots else 'ticket'}}_registration" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-labelledby="staticBackdropLabel" aria-hidden="true">
```

Comment:
Not sure I understand why we have a dynamic ID, seems a bit strange, never seen that before.

Reply:
It's the exact same modal but either we call it first (`modal_ticket_registration`) either we call it second after the slot selection (`modal_slot_ticket_registration`), in the first case we don't need the turnstile and the submit button handling but in the second case we do. Changing the id to apply or not the ModalRegistration interaction class.